swift/extractor/SwiftExtractorConfiguration.h

```diff
@@ -9,7 +9,14 @@ struct SwiftExtractorConfiguration {
   std::string trapDir;
   // The location for storing extracted source files.
   std::string sourceArchiveDir;
-  // The arguments passed to the extractor. Used for debugging.
+  // A temporary directory that exists during database creation, but is deleted once the DB is
+  // finalized.
+  std::string scratchDir;
+  // A temporary directory that contains TRAP files before they moved into the final destination.
+  // Subdirectory of the scratchDir.
+  std::string tempTrapDir;
```

Comment:
unless we use per-process separate `tempTrapDir`, I don't see why we need this to be configurable (and not just the directly the fixed subdirectory of `scratchDir` we currently already have.